src/core/importType.js

```diff
@@ -80,15 +84,16 @@ function isRelativeToSibling(name) {
   return /^\.[\\/]/.test(name)
 }
 
-function typeTest(name, settings, path) {
-  if (isAbsolute(name, settings, path)) { return 'absolute' }
```

Comment:
please revert the unrelated changes here; the extra unused arguments are very intentional.

Reply:
done